packages/crypto-lib/src/base/bignumber-plus.ts

```
@@ -1,25 +1,25 @@
+  let hexStr = value.integerValue().toString(16);
 
-    hexStr = '0x' + hexStr;
-    return hexStr;
+  hexStr = '0x' + hexStr;
+  return hexStr;
 };
 
 const fromBigIntHex = (value: string): BigNumber => {
-    check(value && value.startsWith('0x'), `Invalid hex string. value: ${value}`);
-    return new BigNumber(value).integerValue();
+  check(value && value.startsWith('0x'), `Invalid hex string. value: ${value}`);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)